src/CommunityToolkit.Maui.Core/Essentials/FolderPicker/FolderPickerImplementation.android.cs

```diff
@@ -18,8 +20,16 @@ public async Task<Folder> PickAsync(string initialPath, CancellationToken cancel
 		}
 
 		Folder? folder = null;
+		const string baseUrl = "content://com.android.externalstorage.documents/document/primary%3A";
+		if (Android.OS.Environment.ExternalStorageDirectory is not null)
+		{
+			initialPath = initialPath.Replace(Android.OS.Environment.ExternalStorageDirectory.ToString(), string.Empty, StringComparison.InvariantCulture);
```

Comment:
Let's use the `AbsolutePath` prop. instead of the `ToString` call
```suggestion
			initialPath = initialPath.Replace(Android.OS.Environment.ExternalStorageDirectory.AbsolutePath, string.Empty, StringComparison.InvariantCulture);
```